pkg/cmd/run.go

```diff
@@ -50,7 +49,9 @@ getenvoy run %[1]s -- --help
 			r.Out = c.OutOrStderr()
 			r.Err = c.ErrOrStderr()
 
-			debug.EnableAll(r)
+			for _, err := range debug.EnableAll(r) {
+				fmt.Fprintln(r.Out, "failed to enable option:", err) //nolint
```

Comment:
not sure if this error makes sense for users..

Reply:
before it was "failed to enable debug option" find/replace problem.. I'll put it back!